scripts/examples/nrfconnect_example.sh

```diff
@@ -39,4 +39,9 @@ source "../scripts/activate.sh"
 export GNUARMEMB_TOOLCHAIN_PATH="$PW_PIGWEED_CIPD_INSTALL_DIR"
 env
 
-west build -b "$BOARD" -d "$APP/nrfconnect/build/$BOARD" "$APP/nrfconnect" -- "$@"
+OVERLAY_CONF_FLAGS=()
```

Comment:
please update build_examples.py with similar options.

I just tried:

```
./scripts/build/build_examples.py --target-glob 'nrf*52840*light' build
```

and it seemed to compile fine (it created `nrf-nrf52840-light`). Is the dongle different (assuming one is DK and one is not)? Could we add dongle support to build_examples?

Reply:
Dongle uses the same chipset but lacks some of the auxiliary hardware of the devkit (DK).  It is different enough that it requires different configuration parameters for building.  Would be a good idea to add since the example has been broken for 3 weeks now (since https://github.com/project-chip/connectedhomeip/pull/13031).

Reply:
Thanks for the review! I added the change to build_examples.py to support the dongle. The script didn't attempt to build for the dongle at all so it never failed.

I verified the firmware built from 

```
./scripts/build/build_examples.py --target-glob 'nrf*52840*light' build
```

on a nrf52840dongle.